packages/store/src/internals.ts

```diff
@@ -8,6 +8,18 @@ export interface MetaDataModel {
   children: any[];
 }
 
+/* FROM: https://gist.github.com/jed/982883 */
+/* tslint:disable  */
+export function uuid(a?, b?) {
```

Comment:
Why do we need this? Couldn't we just have a count increment?

```
let count = 0;
getId() { return (++count).toString() };
```

Reply:
yeah I think that would work. would we want one generic helper like this?

Reply:
Sure, a count tracker maybe?

Reply:
Not sure why we would want to use it anywhere else but sure?